src/tests/Regressions/coreclr/GitHub_61104/test61104.csproj

```diff
@@ -9,6 +9,7 @@
 
     <!-- This is an explicit crossgen test -->
     <AlwaysUseCrossGen2>true</AlwaysUseCrossGen2>
+    <NativeAotIncompatible>true</NativeAotIncompatible>
```

Comment:
Is this failing due to https://github.com/dotnet/runtime/issues/101333 ?

Reply:
No, same explanation as for the above. AlwaysUseCrossgen2